google-postgresql.yml

```diff
@@ -26,10 +26,28 @@ plan_updateable: true
 provision:
   plan_inputs:
   user_inputs:
-  - field_name: cores
+  - field_name: tier
     required: true
-    type: number
-    details: Minimum number of cores for service instance.
+    type: string
+    details: Google sql tier. This determines CPU and RAM resource.
+    enum:
+      db-f1-micro: db-f1-micro
+      db-g1-small: db-g1-small
+      db-n1-standard-1: db-n1-standard-1
+      db-n1-standard-2: db-n1-standard-2
+      db-n1-standard-4: db-n1-standard-4
+      db-n1-standard-8: db-n1-standard-8
+      db-n1-standard-16: db-n1-standard-16
+      db-n1-standard-32: db-n1-standard-32
+      db-n1-standard-64: db-n1-standard-64
+      db-n1-standard-96: db-n1-standard-96
+      db-n1-highmem-2: db-n1-highmem-2
+      db-n1-highmem-4: db-n1-highmem-4
+      db-n1-highmem-8: db-n1-highmem-8
+      db-n1-highmem-16: db-n1-highmem-16
+      db-n1-highmem-32: db-n1-highmem-32
+      db-n1-highmem-64: db-n1-highmem-64
+      db-n1-highmem-96: db-n1-highmem-96
```

Comment:
I think there should be a default. Though maybe that's a discussion for another feature.

Reply:
Yes I kept the functionality as it was for cores - there is certainly a case for a default, as there is a default in the Google Console